common/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "common",
```

Comment:
If common is published to npm, i think it's required to have a name we can publish to?

Reply:
This is not going to be published to npm. I've decided to try lerna for the time being to keep the monorepo structure without publishing this. Ultimately the contents of the **dist** folder will be published

Reply:
Tabled monorepo for now